_profiles/transifex/bulk/index.md

```diff
@@ -0,0 +1,396 @@
+    email: kbairak@transifex.com
+
+categories:
+  - Bulk Operations
+---
+
+# Bulk profile
+
+## Introduction
+
+This page specifies a profile for the `application/vnd.api+json` media type,
+as described in the [JSON:API specification](http://jsonapi.org/format/).
+
+This profile allows clients to perform a number of operations with only one
+request. Instead of supporting arbitrary operations on the whole API, this
+profile limits operations of the same type (aka HTTP verb: `POST`, `PATCH`,
+`DELETE`) on the same API resource.
+
+## Creating Resources In Bulk
+
+### Requests
+
+A collection of resources can be created by sending a `POST` request to a URL
+that represents a collection of resources. The request **MUST** include an
+array of resource objects as primary data. All items of the array **MUST** have
+the same `type` member. The same restrictions that apply to a resource object
+when creating a single resource apply to each item of the array. Either all the
+items in the request **MUST** include client-generated IDs, or none **MUST**.
+
+Every bulk create operation **MUST** be atomic. If the server has changed, all
+the requested items **MUST** have been created. Also, no side-effects are to be
+expected to happen asynchronously on the server's side; assuming no further
+interaction with the server, the returned items **SHOULD** remain as they are
+and further attempts to fetch them **SHOULD** return the same resource objects.
+
+For instance, a collection of articles might be created with the following
+request:
+
+```
+POST /articles HTTP/1.1
+Content-Type: application/vnd.api+json;profile="https://github.com/json-api/json-api/_profiles/transifex/bulk/index.md"
+Accept: application/vnd.api+json;profile="https://github.com/json-api/json-api/_profiles/transifex/bulk/index.md",application/vnd.api+json
+
+{
+  "data": [
+    {
+      "type": "articles",
+      "attributes": {"title": "Creating resources in bulk"}
+    },
+    {
+      "type": "articles",
+      "attributes": {"title": "Creating many resources at the same time"}
+      "relationships": {"author": {"data": {"type": "users", "id": "1"}}}
+    }
+  ]
+}
+```
+
+### Responses
+
+#### `201 Created`
+
+If the requested items did not include Client-Generated IDs and the requested
+resources have been created successfully, the server **MUST** return a
+`201 Created` status code.
+
+The server **MUST NOT** include a `Location` header.
+
+The response **MUST** also include a document that contains an array with the
+primary resources created, in the same order as they appear in the request.
+
+For instance, A response when all requested items were created can be:
+
+```
+201 Created
+{
+  "data": [
+    {
+      "type": "articles",
+      "id": "1",
+      "attributes": {"title": "Creating resources in bulk"}
+    },
+    {
+      "type": "articles",
+      "id": "2",
+      "attributes": {"title": "Handling responses in bulk creation"}
+    }
+  ],
+  "links": {
+    "profile": [
```

Comment:
Per latest v1.1 release candidate, applied profiles and extensions should be listed in `jsonapi` object. Not in `links`. See https://jsonapi.org/format/1.1/#document-jsonapi-object